### ISSUE TITLE
Feature Request: Support separate H and V TTL sync inputs

### ISSUE THREAD

**Issue by maxtherabbit** [skumlos/bkm-68x-simple#1]
This feature would be helpful for the Dreamcast, as well as a myriad of retro computers that output separate sync. Judging by the existing design where all sync sources are separated before being passed along to the monitor, it seems like a fairly trivial feature to accommodate. Amazing work so far!

**Comment by skumlos** [comment]
Yes, this should be trivial besides whatever switching mechanism that would also be "needed" and I will definitely put something like this into this or a future version. I dunno how much it directly helps Dreamcast if the sync from it is whack (as implied by the need of a multivibrator circuit in the vblank period as I understand is present in the RetroAccess cable) but I am gonna look into some of that soon :) and thanks!

**Comment by maxtherabbit** [comment]
Well I'm no expert on Dreamcast sync, however I do believe that the use of
the multivibrator in RA's cable is only needed to recreate the serration
pulses during v-blank.

As shown in the attached scope captures, you can see that when natively
outputting separate HV sync, the dreamcast syncs are quite "well-behaved"
with proper h-sync pulses throughout v-blank

**Comment by maxtherabbit** [comment]
![dc15](https://user-images.githubusercontent.com/6538577/126727915-374dc4dd-2465-4109-b2ef-30fe085de979.png)
![dc31](https://user-images.githubusercontent.com/6538577/126727918-673cf2c6-5ea5-4380-8a76-8db2ff13eb3f.png)
